atd-vze/src/views/Crashes/Recommendations/Recommendations.js

```diff
@@ -74,7 +75,7 @@ const Recommendations = ({ crashPk, recommendation, refetch }) => {
   const onAddPartner = valuesObject => {
     const recommendationPartnerRecord = {
       recommendationRecordId,
-      ...valuesObject,
+      ...valuesObject.recommendations_partners.data,
```

Comment:
dont want that nested insert format if the recommendation record already exists and we are only inserting a partner record